eth/vm/forks/byzantium/__init__.py

```diff
@@ -80,9 +92,3 @@ def validate_receipt(cls, receipt: Receipt) -> None:
     @staticmethod
     def get_block_reward():
         return EIP649_BLOCK_REWARD
-
-    @staticmethod
-    def get_uncle_reward(block_number, uncle):
```

Comment:
Alternatively, we could make this a `@classmethod` and then access the block reward on `cls.block_reward` which would allow us drop any uncle reward refinement in `ConstantinopleVM`. Thoughts?

Reply:
Conceptually I'm fine with that but I think it goes against the direction we're trying to go with composition > inheritance.  I like the mechanism you've got in place now with the curried re-usable utilities so if you're happy with that I say keep it (and maybe backport it to the `ByzantiumVM`)

Reply:
Yup, I like it as-is, too. I kind of like the idea of pushing the `get_uncle_reward(...)` definition as far back as possible, so that even the import carries semantic meaning, like:

```py
from eth.vm.forks.frontier import get_uncle_reward
```

Would mean that the uncle mechanism has stayed the same since frontier, with only the block reward changing in: `get_uncle_reward = staticmethod(get_uncle_reward(EIP649_BLOCK_REWARD))`.

(I don't remember when/if the uncle mechanism changed, but the general concept stands.)

Reply:
SGTM :+1:

Reply:
>Yup, I like it as-is, too. I kind of like the idea of pushing the get_uncle_reward(...) definition as far back as possible

It was last changed in `byzantium` so I think it is already pushed as far back as possible.